src/apis/register/registerApi.ts

```
@@ -0,0 +1,15 @@
+import axios from "@/apis/instance";
+
+export const registerApi = async (nickname: string) => {
+  try {
+    const response = await axios.post("/api/v1/auth/register-anonymous", {
+      nickname,
+    });
+    localStorage.setItem("NapulNapul-accessToken", response.data.accessToken);
+    localStorage.setItem("NapulNapul-refreshToken", response.data.refreshToken);
+    return true;
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)